packages/plugin-stage/sources/drivers/GitDriver.ts

```diff
@@ -90,15 +90,29 @@ async function genCommitMessage(cwd: PortablePath, changes: Array<stageUtils.Fil
   return message;
 }
 
+const unstagedPrefixes = {
+  [stageUtils.ActionType.CREATE]: [` A `, `?? `],
+  [stageUtils.ActionType.MODIFY]: [` M `],
+  [stageUtils.ActionType.DELETE]: [` D `],
+};
+
+const stagedPrefixes = {
+  [stageUtils.ActionType.CREATE]: [`A  `],
+  [stageUtils.ActionType.MODIFY]: [`M  `],
+  [stageUtils.ActionType.DELETE]: [`D  `],
+};
+
 export const Driver = {
   async findRoot(cwd: PortablePath) {
     return await stageUtils.findVcsRoot(cwd, {marker: `.git` as Filename});
   },
 
-  async filterChanges(cwd: PortablePath, yarnRoots: Set<PortablePath>, yarnNames: Set<string>) {
+  async filterChanges(cwd: PortablePath, yarnRoots: Set<PortablePath>, yarnNames: Set<string>, staged = false) {
```

Comment:
As convention we prefer to avoid boolean arguments:
```suggestion
  async filterChanges(cwd: PortablePath, yarnRoots: Set<PortablePath>, yarnNames: Set<string>, {staged = false}: {staged?: boolean} = {}) {
```

Reply:
That is a good convention 🙂 
Fixed.